fili-sql/src/main/resources/moduleConfig.properties

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Yahoo Inc.
+# Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+#The name of this module (required)
+moduleName = fili-sql
+
+moduleDependencies = fili-core
+
+# Disabling the druid dimension loader by default
+# When set to true you will also need to populate the list of dimensions using druid_dim_loader_dimensions
+bard__druid_dimensions_loader_enabled = false
+bard__druid_dim_loader_row_limit = 1000
```

Comment:
These aren't relevant until later in the rfc, should I take it out @michael-mclawhorn ?